source/fundamentals/data-formats/codecs.txt

```diff
@@ -39,4 +39,11 @@ BsonTypeClassMap
 Custom Codec Example
 --------------------
 
-- Example, perhaps something like this https://stackoverflow.com/questions/45083885/decode-document-to-a-java-class-using-custom-mongo-codec
\ No newline at end of file
+- Methods to override
+- Encode: Describe parameters: BsonReader, EncoderContext
+- Decode: Describe parameters: BsonReader, DecoderContext
+- Example, perhaps something like this https://stackoverflow.com/questions/45083885/decode-document-to-a-java-class-using-custom-mongo-codec
+- Sample results from custom codec example
+- Explain one or more commonly-encountered exceptions such as BsonSerializationException
+- Example showing how to read/encode a document with unknown fields
+- Table showing type mappings and corresponding methods
```

Comment:
Does this belong in the custom codec example?

Reply:
I'm starting to lose context on what this was meant to be. I think it was BsonValue sub-type mappings, but not certain why I did not place it under Document > BsonDocument instead. I'll remove it for now.

Reply:
Did you remove it?

Reply:
From both my last commit, and preview, I did remove it: https://github.com/mongodb/docs-java/pull/35/commits/583e2c87430e3f3e2c43b2fcea9d889b085d60da#diff-dda4e6ecdf7c59166b009c08afae23ebL49

Reply:
No, I was just expecting a "Done" comment to indicate it.